target_game.py

```diff
@@ -23,23 +26,119 @@ def move_target_at_random_pos(target):
     new_pos = (pos_x, pos_y)

```

Comment:
as I said before, I am really confused about these @signs and why they hide the functions! But anyways. A thing that I noticed is that you define some functions before the welcome screen. Is there a logic behind the order of your functions?

Reply:
Yes, I firstly define the functions as I don't want them to be part of the program - in this way I can easily use them in my loops.